### PR TITLE
Ratings modal

### PR DESCRIPTION
**Pull request by Jasper-Nelligan**
Adds a ratings modal that's activated when the printer clicks "Exchanged" in the order status.

The rating is then sent to the backend. With these ratings we can create the reviews section on the profile page

Screenshot (58)

### CODE REVIEW

**Comment by jvsluis** [comment]
This works well on my end. However, this is only unidirectional. This allows the printer to rate the printee (since they are the ones switching the status to exchanged). We'll also need a way for the printee to rate the printer.

Maybe after the printer sets the status to exchanged, the printer does their review and the job is removed from their account. Then if the printee views the page after being set to exchanged and they havent reviewed yet, they get the option to review the printer. Once the printee has reviewed then the order is no longer listed on their account.

**Comment by Jasper-Nelligan** [comment]
@jvsluis I changed the rating system to use stars instead of numbers, just as a heads up

File: src/components/ratingModal.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  const onSubmit = () => {
+    const overallRatingNumber = getOverallRatingNumber();
+    const date = getDate();
+    let averageRating, rating;
+    if (isCustomer) {
+      averageRating = (overallRatingNumber + selectedStarPrintQuality + selectedStarCommunication + selectedStarExchange) / 4;
+      rating = {
+        targetUserUid,
+        averageRating,
+        printQualityRating: selectedStarPrintQuality,
```
Comment (mas393) [inline src/components/ratingModal.js:106]: Why not upload overall rating as well?

**Review by mas393 — APPROVED** [review]
I can add ratings & comments as expected and I see them uploaded to firebase

**Pull request merged**